test/setup.js

```diff
@@ -168,3 +186,13 @@ function extractImportantFields (parsedTorrent) {
   const priv = parsedTorrent.private // private is a reserved word in JS
   return { infoHash, name, announce, urlList, comment, 'private': priv }
 }
+
+function copy (pathFrom, pathTo) {
+  try {
+    fs.copySync(pathFrom, pathTo)
+  } catch (e) {
+    // Windows sometimes gives us an EPERM error even tho the copy happened...
+    if (process.platform !== 'win32' || e.code !== 'EPERM') throw e
+    console.log('ignoring windows copy EPERM error', e)
```

Comment:
This is janky. Are we sure there's not an open file handle at the path we are copying to? That could cause this error. See https://github.com/nodejs/node-v0.x-archive/issues/6599#issuecomment-30089704